lib/parseCommandLineOptions.js

```diff
@@ -26,6 +26,23 @@ module.exports = function parseCommandLineOptions(argv) {
       type: 'string',
       demand: false,
     })
+    .options('browsers', {
+      describe:
+        "Specify which browsers to support. Controls which font formats to provide, and whether to apply a JavaScript-based polyfill for preloading fonts. Defaults to browserslist's default query: https://github.com/browserslist/browserslist#best-practices",
+      type: 'string',
+      demand: false,
+    })
+    .options('formats', {
+      describe:
+        'Font formats to use when subsetting. The default is to select the formats based on the browser capabilities as specified via --browsers or the browserslist configuration.',
+      type: 'array',
+      choices: ['woff2', 'woff', 'truetype'],
+    })
+    .options('js-preload', {
```

Comment:
I'm not sure we should expose this option. I'd rather promote usage of browserslist project configuration to have the tool just do the right thing

Reply:
Hmm, yeah, I agree. I've taken it out in dccddcf